index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="css/styles.css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="header-item" id="logo-container">
+      <a href="#">John Doe</a>
+    </div>
+    <div class="header-item">
+      <nav>
+        <button class="header-item" id="menu-button">
+          <img src="icons/Icon-Menu.png" alt="btn">
+        </button>
+      </nav>
+    </div>
+  </header>
+
+  <section class="black-bg" id="headline">
+    <div class="bckgrnd right">
+      <img src="img/Illustration.png" alt="">
+    </div>
+
+    <div class="container column">
+      <h1>Hey There. I’m Alex</h1>
+      <h3>I’am a Software Developer</h3>
+      <p class="wht texto1">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <div>
+        <img class="icon" src="icons/Icon-GitHub.png" alt="Icon-GitHub">
+        <img class="icon" src="icons/Icon-Linkedin.png" alt="Icon-Linkedin">
+        <img class="icon" src="icons/Icon-Angelist.png" alt="Icon-Angelist">
+        <img class="icon" src="icons/Icon-Twitter.png" alt="Icon-Twitter">
+        <img class="icon" src="icons/Icon-Medium.png" alt="Icon-Medium">
+      </div>
```

Comment:
- [ ] Kindly, replace the `div` with `ul` and put your icons `img` inside a `li` as requested by the previous reviewer.
- [ ] Your ions are not displaying, please fix your images links.